File: PHP/account.php
<?php
class Account {
    public $id = integer;
    public $name      = string;
    public $document  = string;
    public $email     = string;
    public $password  = string;

    public function __construct($name, $document, $email) {
        $this->name = $name;
        $this->document = $document;
        $this->email = $email;
    }

    public function PrintDataAccount(){
        echo "Name: $this->name, Document: {$this->document}, document: {$this->email}";
    }
}
?>

File: Java/Account.java
public class Account {
    Integer id;
    String name;
    String document;
    String email;
    String password;

    public Account(String name, String document, String email){
        this.name = name;
        this.document = document;
        this.email = email;
    }
    void printDataAccount() {
        System.out.println("Name: " + name);
        System.out.println("Document: " + document);
        System.out.println("Email: " + email);
    }
}
